src/index.ts

```diff
@@ -54,10 +54,10 @@ export class Store extends EventEmitter {
     return Object.values(this.store);
   }
 
-  stream(): PassThrough {
+  stream(filter: (file: File) => boolean = () => true): PassThrough {
```

Comment:
Any thoughts on a function directly VS an option object? Is there a standard API we can based ourselves on?

Reply:
I cannot think in any use of an option object right now.
If you prefer, ok to me.

The filter as a direct parameter looks a bit more simpler and the stream is so simple that in any other case may need an alternative implementation.
The filter parameter itself is not really required but doing a pipe with the same filter in the stream is taking 6s (< 1s with filter param) and runs 3 times, and a few more is planned.

Reply:
Thanks!

Let's go with an option object; I'd feel better knowing it can be extended without introducing a breaking change.

Reply:
Done